### DISCUSSION TITLE
Crypto not deposited to my external wallet

### DISCUSSION THREAD

**Discussion by Rosedav** [blockscout/blockscout#11056] [General]
What's happening, is there a delay of wld coins deposits to external address

**Comment by IVODICARLO** [comment]
Que se passe-t-il, y a-t-il un retard dans les dépôts de pièces de monnaie sauvages vers une adresse externe

**Reply by IVODICARLO** [reply]
J'ai ce problème depuis le 25/10/2024

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
